src/components/ProductCard/ProductCard.scss

```diff
@@ -0,0 +1,116 @@
+* {
+  margin: 0;
+}
```

Comment:
Well, I'm not sure about this rule here)
try to use it in `index.scss` if it works for you would be great because later we could meet some problems with a styles overwriting, and we won't know where is this rule)

else styles i loke so much!) :fire: